fast_api_als/routers/submit_lead.py

```diff
@@ -55,6 +56,7 @@ async def submit(file: Request, apikey: APIKey = Depends(get_api_key)):
         }
         item, path = create_quicksight_data(obj, 'unknown_hash', 'REJECTED', '1_INVALID_XML', {})
         s3_helper_client.put_file(item, path)
+        logging.error("[Submit Lead] XML Rejected. Code - 1_INVALID_XML.\nThe XML -\n%s",body)
```

Comment:
using `[Submit Lead]` in every log statement is a bad practice.

Reply:
I have fixed it. Changed the formatting of the logs.